src/pages/Architecture.tsx

```
@@ -0,0 +1,261 @@
+          >
+            {darkMode ? <Sun className="w-5 h-5" /> : <Moon className="w-5 h-5" />}
+          </button>
+        </div>
+      </div>
+
+      {/* 1. Introduction */}
+      <div className="doc-section-card mb-12">
+        <h2 className="text-2xl font-semibold mb-2">Introduction</h2>
+        <div className="documentation-markdown text-muted-foreground mb-4">
+          <ReactMarkdown
+            children={projectDoc.description || 'No project description available.'}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)